website/static/js/quickProjectSearchPlugin.js

```diff
@@ -37,20 +37,19 @@ var quickSearchProject = {
                 self.nodes().push(node);
                 self.retrieveContributors(node);
             });
-            self.populateEligibleNodes(0, self.countDisplayed() - 1);
+            self.populateEligibleNodes(0, self.countDisplayed());
             self.next(result.links.next);
         });
         promise.then(
             function(){
-                self.recursiveNodes(self.next());
+                if (self.next()) {
+                    self.recursiveNodes(self.next());
```

Comment:
This looks weird, to have two `self.next()` calls one right after the other. My gut says that we're accidentally throwing away the result of the first call. Upon investigation, it's clear that we're not (`self.next` [is](https://github.com/CenterForOpenScience/osf.io/blob/03034779d421991e46f85a8ef1af3b07306228a8/website/static/js/quickProjectSearchPlugin.js#L24) a Mithril [prop](http://mithril.js.org/mithril.prop.html), which returns the same value from successive calls until it is called with a new value). I suppose we can assume familiarity with Mithril here, and I will retrain my gut. :-)